command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
Change:
```
A conformant [runtime][] MUST provide an executable...
```
to something like:
```
A compliant implementation of this "command line interface" specification MUST provide an executable...
```
To make it clear that we're talking about being compliant with the CLI spec, not the RUNTIME spec.